vignettes/pavo.Rmd

```diff
@@ -54,31 +54,24 @@ The first thing we need to do is import the spectral data into R using the funct
 
 The files were downloaded and placed in a directory called `/pavo/vignette_data`. By default, `getspec` will search for files in the current folder, but a different one can be specified:
 
-```{r, echo=FALSE, eval=TRUE, results='hide', include=FALSE}
-# specs <- getspec(system.file("extdata", package = "pavo"), ext = "ttt", decimal = ",", subdir = TRUE, subdir.names = FALSE)
-specs <- readRDS(system.file("extdata/specsdata.rda", package = "pavo"))
+```{r, results='hide', include=TRUE}
+specs <- getspec("../vignette_data/", ext = "ttt", decimal = ",", subdir = TRUE, subdir.names = FALSE)
 ```
 
-```{r, echo=TRUE, eval=FALSE, results='hide', include=TRUE}
-#specs <- getspec("~/pavo/vignette_data/", ext = "ttt", decimal = ",", subdir = TRUE, subdir.names = FALSE)
-# 213  files found; importing spectra
-# |================================================================================| 100%, ETA 00:00
-```
-
-```{r, echo=TRUE, eval=TRUE}
+```{r}
```

Comment:
```{r, echo=TRUE, eval=FALSE, results='hide', include=TRUE}```
```#specs <- getspec("~/pavo/vignette_data/", ext = "ttt", decimal = ",", subdir = TRUE, subdir.names = FALSE)
 # 213  files found; importing spectra
 # |================================================================================| 100%, ETA 00:00
 ```

Yeah this was to avoid adding the ~1.5mb of vignette_data specs to the package install, so we 'faked' it by showing the user what it will look like, while loading the ~300kb rds file in the background. Not sure if there's a more elegant solution? I suppose I could explain this in the text instead ("we would normally do x, which would look like this, but here we'll do y").